.github/workflows/pytest-gpu.yml

```diff
@@ -66,6 +66,13 @@ jobs:
             pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoelastic/viscoelastic_example.py
         fi
 
+    - name: Run with MPI
+      if: matrix.name == 'pytest-gpu-acc'
+      run: |
+        DEVITO_MPI=1 mpirun -n 2 py.test examples/seismic/elastic/elastic_example.py
+        DEVITO_MPI=1 mpirun -n 2 py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
```

Comment:
I'd rather have TTI here, viscoacoustic not much different  than viscoelastic

Reply:
we now do have tti above? can we resolve this?